tests/cts/scripts/run_cache_sparql.py

```diff
@@ -2,7 +2,7 @@
 import sys
 
 sys.path.append(
-    os.path.join(os.path.dirname(__file__), "..", "..")
+    os.path.join(os.path.dirname(__file__), "..", "..", "..")
 )
 
 from capitains_nautilus.cts.resolver import SparqlAlchemyNautilusCTSResolver
```

Comment:
`SparqlAlchemyNautilusCTSResolver` should probably be `SparqlAlchemyNautilusCtsResolver`